src/stabilization_guide.md

```diff
@@ -0,0 +1,181 @@
+
+# Request for stabilization
+
+    Once an unstable feature has been well-tested with no outstanding
```

Comment:
Could you please un-indent this paragraph? It renders as a code block. (also applies for a few other places below).

Reply:
done